website/common/script/cron.js

```diff
@@ -88,37 +91,52 @@ export function daysSince (yesterday, options = {}) {
   Should the user do this task on this date, given the task's repeat options and user.preferences.dayStart?
  */
 
-export function shouldDo (day, dailyTask, options = {}) {
+export function shouldDo (day, dailyTask) {
```

Comment:
If the `options` param is removed from the callee, then it should be removed from all callers as well.